app/assets/javascripts/discourse/tests/acceptance/composer-test.js

```diff
@@ -625,22 +625,22 @@ acceptance("Composer", function (needs) {
     );
 
     await menu.expand();
-    await menu.selectRowByValue("toggleWhisper");
+    await menu.selectRowByName(I18n.t("composer.toggle_whisper"));
 
     assert.ok(
       !exists(".composer-actions svg.d-icon-far-eye-slash"),
       "it removes the whisper mode"
     );
 
     await menu.expand();
-    await menu.selectRowByValue("toggleWhisper");
+    await menu.selectRowByName(I18n.t("composer.toggle_whisper"));
```

Comment:
The value here has changed so using `name` to select. Shouldn't be an issue -- the `value` before was sometimes a function sometimes a string so I would argue this is actually more consistent.